style.css

```diff
@@ -1,4 +1,4 @@
-﻿.x-body .layer-selector2 {
+.x-body .layer-selector2 {
```

Comment:
I know github is telling me these lines are different, but I swear I can't see it.  Looks like the ext class is still there?